testsupport/space.go

```diff
@@ -205,14 +208,11 @@ func verifyResourcesProvisionedForSpace(t *testing.T, hostAwait *wait.HostAwaiti
 		require.NoError(t, err)
 		require.Contains(t, ns.Labels, toolchainv1alpha1.WorkspaceLabelKey)
 		assert.Equal(t, space.Name, ns.Labels[toolchainv1alpha1.WorkspaceLabelKey])
-
-		// check that namespace list is available in the NSTemplate status
-		nsTmplSet, err = targetCluster.WaitForNSTmplSet(t, spaceName, wait.UntilNSTemplateSetHasProvisionedNamespaces([]toolchainv1alpha1.SpaceNamespace{{
-			Name: spaceName + "-tenant",
-			Type: "default", // default type should be there
-		}}))
-		require.NoError(t, err)
 	}
+	// wait for space to have list of provisioned namespaces in status
+	_, err = hostAwait.WaitForSpace(t, spaceName,
+		wait.UntilSpaceHasProvisionedNamespaces(nsTmplSet.Status.ProvisionedNamespaces))
```

Comment:
Let's add a comment here to mention the nsTmplSet ProvisionedNamespaces is checked as part of VerifyNSTemplateSet?

Reply:
makes sense! done [a7ebfe3](https://github.com/codeready-toolchain/toolchain-e2e/pull/667/commits/a7ebfe353db58e3c285fc10f9579800bd91e5401)

Thanks!